src/RedditPosts.js

```diff
@@ -1,153 +1,126 @@
-import React, { Component } from 'react';
-import RedditPostIt from './RedditPostIt';
-import RedditListing from './RedditListing';
-import RedditTile from './RedditTile';
+import React, { Component } from "react";
```

Comment:
i always get confused on when to use single quotes and double quotes. 
in javascript and python!

Reply:
Apologies for the change from single to double - it 100% doesn't matter in JS! My formatter (prettier) uses double quotes by default. Use whatever tickles your fancy